applications/crossbar/src/crossbar_doc.erl

```diff
@@ -633,14 +650,15 @@ update(Context, DocId, Updates, Creates) ->
                     ,{'create', Creates}
                     ,{'ensure_saved', 'true'}
                     ],
-    case kz_datamgr:update_doc(cb_context:account_db(Context), DocId, UpdateOptions) of
+    {Context1, WaitFor} = extract_wait_for_service_update(Context),
+    case kz_datamgr:update_doc(cb_context:account_db(Context1), DocId, UpdateOptions) of
```

Comment:
I'm not sure how should be done here but you have to delete `wait_for_service_update` from UpdateOptions too (from both `Created` and `Updates` parameters).